app/controllers/green_list_controller.rb

```diff
@@ -93,6 +98,10 @@ def find_protected_area
     @protected_area or raise_404
   end
 
+  def get_green_list_sites
+    @example_greenlist ||= ProtectedArea.green_list_areas.take(3)
```

Comment:
@stanleypliu I guess you could use the `green_list_areas` method below instead of rewriting the same `ProtectedArea.green_list_areas` logic?